### ISSUE TITLE
INEL corpora upload script

### ISSUE THREAD

**Issue by myrix** [ispras/lingvodoc-react#432]
We need an automated script for uploading processed INEL Dolgan, Kamas and Selkup corpora to a Lingvodoc server.

**Comment by myrix** [comment]
Implemented, tested locally.

NOTE for testers: for testing the script on test stand http://83.149.198.62/ updated DB dump there to the latest one, from 2021-02-26.16-51, this can possibly lead to unrelated test fails.

**Comment by myrix** [comment]
Script invocations for Dolgan, Kamas and Selkup respectively:
```
python -m lingvodoc.scripts.exmaralda_to_eaf -config $LINGVODOC_DIR/development.ini -no-db upload_corpus $CORPUS_DIR/dolgan-corpus.zip $DOLGAN_OUTPUT_DIR --graphql-url http://lingvodoc.ispras.ru/api/graphql --cookies-dict '{"locale_id": "2", "auth_tkt": "$AUTH_COOKIE", "client_id": "$CLIENT_ID_COOKIE"}' --language-id '[1105, 28]' --language-name Dolgan
python -m lingvodoc.scripts.exmaralda_to_eaf -config $LINGVODOC_DIR/development.ini -no-db upload_corpus $CORPUS_DIR/kamas.zip $KAMAS_OUTPUT_DIR --graphql-url http://lingvodoc.ispras.ru/api/graphql --cookies-dict '{"locale_id": "2", "auth_tkt": "$AUTH_COOKIE", "client_id": "$CLIENT_ID_COOKIE"}' --language-id '[1552, 652]' --language-name Kamas
python -m lingvodoc.scripts.exmaralda_to_eaf -config $LINGVODOC_DIR/development.ini -no-db upload_corpus $CORPUS_DIR/selkup.zip $SELKUP_OUTPUT_DIR --graphql-url http://lingvodoc.ispras.ru/api/graphql --cookies-dict '{"locale_id": "2", "auth_tkt": "$AUTH_COOKIE", "client_id": "$CLIENT_ID_COOKIE"}' --language-id '[508, 51]' --language-name Selkup --dialect-coma $CORPUS_DIR/selkup.coma
```
Source corpus files, including metadata file `selkup.coma`, are available for download from links in #400.

NOTE for testers: not for automatic testing.

**Comment by nadezhday** [comment]
Closing without verification/
